### ISSUE TITLE
Remove babel-polyfill dependency

### ISSUE THREAD

**Issue by damianobarbati** [oliver-moran/jimp#748]
# Current Behavior

Having jimp as a dependency results now in the following *warning* when running `yarn install`:
```
warning html5-webpack-plugin > favicons > jimp > @babel/polyfill@7.4.4: 🚨 As of Babel 7.4.0, this
package has been deprecated in favor of directly
including core-js/stable (to polyfill ECMAScript
features) and regenerator-runtime/runtime
(needed to use transpiled generator functions):

  > import "core-js/stable";
  > import "regenerator-runtime/runtime";
```

**Comment by XhmikosR** [comment]
Can someone make a patch about this?

**Comment by pi0** [comment]
If you are using jimp **for Node.js** (not browser) you may try [jimp-compact](https://github.com/nuxt-community/jimp-compact) a repackaged version which is much smaller and also hasn't a dependency on jimp.

**NOTE:** This is **NOT** an official solution.

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in v0.6.8 :rocket:
